docs/UserGuide.md

```diff
@@ -495,35 +525,33 @@ Perform actions on a group of contacts.
 
 Format: `range COMMAND from/INDEX_FROM to/INDEX_TO`
 
-* Performs the specified `COMMAND` on all contacts between the specified range of `INDEX_FROM` to `INDEX_TO` inclusive
-* `COMMAND` must be a valid command. The allowed operations in `COMMAND` are
-  * list
+* Performs the specified `COMMAND` on all contacts between the specified range of `INDEX_FROM` to `INDEX_TO` inclusive.
+* `COMMAND` must be a valid command without `INDEX`. The allowed operations in `COMMAND` are:
   * edit
   * delete
-* The `INDEX_FROM` and `INDEX_TO` parameters must be **positive integers**, and refer to the index number shown in the **displayed contact list**
-* `INDEX_FROM` must be less than `INDEX_TO` must be supplied, otherwise the command will perform no operation
-* The resultant effect of the command is dependent on the performed action
+* The `INDEX_FROM` and `INDEX_TO` parameters must be **positive integers**, and refer to the index number shown in the **displayed contact list**.
```

Comment:
This should be displayed persons list to be consistent with all other commands.

Reply:
thx changed